Microsoft.Research/Clousot.Cache/Models/Mapping/SuggestionMap.cs

```diff
@@ -30,7 +30,7 @@ public SuggestionMap()
             // Properties
             this.Property(t => t.Kind)
                 .IsRequired()
-                .HasMaxLength(100);
+                .HasMaxLength(400);
```

Comment:
Just in case, can we extract this into the constant with small comment that will describe why this size should be used?